.github/workflows/build.yml

```diff
@@ -76,29 +76,34 @@ jobs:
         if: startsWith(matrix.os, 'windows')
         run: echo "C:\Program Files\Conan\conan\" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
 
+      - name: Enable Developer Command Prompt
+        if: startsWith(matrix.os, 'windows')
+        uses: ilammy/msvc-dev-cmd@v1.12.0
```

Comment:
Move this back to under `Install dependencies` unless you figure out this is actually necessary